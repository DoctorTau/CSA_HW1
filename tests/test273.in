654
6230 -3008 -524 -4766 8043 -8156 -4159 -9043 -4136 -7474 4475 -1890 6187 -6786 -4251 4115 -226 -3237 9468 -6702 -5103 -4339 -6998 -7551 2579 -1879 -3711 2836 8432 7290 6586 -6714 -6925 -4867 -4203 -2515 -1782 9665 -9794 -2057 -6171 -7414 9658 8128 -1250 3568 3098 8370 9819 4918 -6519 6161 989 6094 -4871 -3626 -9525 4333 8394 -8330 3213 3741 -8892 -8072 -525 6940 515 7586 3586 -4720 -5069 -7362 -3891 -8786 4892 1808 5112 -8075 8136 -3134 166 3114 -4563 -5407 -4739 -2701 5496 1563 -9957 -5467 496 -3258 1037 -4568 9459 4911 -3341 -3694 1025 -9203 3991 -573 -2783 1193 -8808 2283 -1472 -2449 9737 2470 -6574 9265 -2660 2682 458 82 -4071 -3699 -1958 -5481 4001 -1842 122 -1855 -9948 9302 9578 4055 4009 6597 -3045 5819 7498 1156 5777 657 -2987 2530 -9741 -9863 1608 3228 7748 -3243 -4794 -9926 -1987 2916 7361 -6377 9325 -5352 -2570 -1361 4903 -5409 27 -4824 -2237 -8670 -2776 1290 -8463 -4793 -5692 3916 2470 5210 -4002 -2198 -6050 9887 2943 -7560 -8289 478 -7137 -4718 -6344 -1135 -6650 -5143 7016 -7811 1075 3424 3088 1573 3408 -9903 5930 5829 -5482 5125 -7037 -3129 4669 -785 -6204 -7124 -1126 2144 -1523 -3445 5085 6738 3807 -8451 -6741 -2336 3471 -7909 -4733 9805 1214 61 562 310 -5810 324 2825 7928 2812 9914 -6397 -515 6651 6213 3657 3924 6223 7360 -6296 -230 5061 1139 -4730 -1114 1131 2911 -3178 3527 -7985 -166 6974 7616 4057 8519 5602 -9798 -7370 -1902 -743 9586 -7311 -8927 -5453 6674 1069 -7918 460 3257 6407 6240 7690 4416 4611 -4129 -686 6561 6530 4847 9334 -7018 6791 5853 5586 -4093 -7657 -1659 -3616 9670 -725 -169 6786 -62 2757 9897 2187 8798 -8149 -1783 6432 -9120 -8227 -9595 7857 7987 -1046 -7449 -7760 -9191 -4649 8795 2896 2417 -9728 -7499 8626 6310 4404 -5762 -4309 6930 -681 3073 -5867 3039 -9985 -9871 8203 2844 -9928 8890 9362 -7715 -1673 -3283 -649 -5438 7980 -2173 5247 3906 9067 994 5590 8679 6118 7843 -7879 6493 -5393 -4515 -8100 1447 -7911 -3839 -8598 -2636 -8484 3987 -8787 795 9734 4312 -703 -6059 -3135 -6142 6833 550 -4761 8559 -7206 3480 5467 3228 7458 -7731 287 2379 9254 7105 5221 -4782 -4260 1842 -7169 7405 -3009 239 -7474 -1348 -9022 -8367 -841 -2398 9271 8809 7947 4696 7102 6054 7111 6729 -9080 3657 -7698 5089 -4431 -3772 5352 1745 9575 2328 -4223 9225 -5111 6745 4078 3494 650 -1078 -7696 3946 6269 8183 -557 -9977 -4937 9899 -8436 393 -7053 -1940 -3568 42 8156 3717 -3817 3993 1220 -2650 3807 1829 8330 8867 306 -8745 1495 -476 4538 2282 7949 -9658 -4670 3034 -3896 1051 4799 4121 -9022 2362 -5913 868 -3602 -2480 -2542 -1158 406 -6777 -2439 3721 -5524 3535 9360 -2807 2239 389 7570 -6879 -2234 -4207 -7067 -5338 -5708 -4234 -7946 -7194 1243 8917 -2703 -8759 -3658 8691 3888 -7893 8087 -281 7041 7132 261 -2194 8596 9315 -2261 -8793 3823 -5695 -3622 502 8682 -7443 -3408 2182 -4745 -2102 -5876 -2538 340 -2885 9210 906 -9815 -922 -3156 9485 -9850 1684 6387 2004 -9236 -938 9242 1211 -7906 5783 2179 9415 -1678 -6402 -8154 9436 6711 -1036 3547 4862 6327 -9423 -2950 6198 -2967 -8417 5279 -5680 -4242 -9779 -2088 -5470 3090 1524 1729 8148 -648 9760 -652 -2910 8732 7402 -895 -6511 1701 7592 -3642 -1725 -7531 -4742 -7936 176 -8343 4327 4068 8727 6490 918 -5931 4175 999 -6576 1276 812 -8117 8667 -4628 -3315 7761 2688 -1444 -5951 8683 -6844 -9204 9344 -7837 -5126 -2376 1389 5853 9301 3532 -3195 2116 -2621 -4157 4953 -1348 -8835 5492 8457 6897 -7689 -5150 2148 -4794 9605 65 5511 -3124 8580 -5185 5847 5462 2727 -4590 9789 4814 -245 3463 -332 -6723 -6134 8703 -4080 8653 -8474 8651 -8727 2487 3857 -3498 8129 2861 3951 -2420 1457 -5643 3620 6546 -3935 -7651 -6840 -3251 -3618 