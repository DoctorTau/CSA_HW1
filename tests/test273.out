-3008 -524 -4766 -8156 -4136 -7474 -1890 -6786 -226 -6702 -6998 -6714 -1782 -9794 -7414 -1250 -3626 -8330 -8892 -8072 -4720 -7362 -8786 -3134 -3258 -4568 -3694 -8808 -1472 -6574 -2660 -1958 -1842 -9948 -4794 -9926 -5352 -2570 -4824 -8670 -2776 -5692 -4002 -2198 -6050 -7560 -4718 -6344 -6650 -5482 -6204 -7124 -1126 -2336 -5810 -6296 -230 -4730 -1114 -3178 -166 -9798 -7370 -1902 -7918 -686 -7018 -3616 -62 -9120 -1046 -7760 -9728 -5762 -9928 -5438 -8100 -8598 -2636 -8484 -6142 -7206 -4782 -4260 -7474 -1348 -9022 -2398 -9080 -7698 -3772 -1078 -7696 -8436 -1940 -3568 -2650 -476 -9658 -4670 -3896 -9022 -3602 -2480 -2542 -1158 -5524 -2234 -5338 -5708 -4234 -7946 -7194 -3658 -2194 -3622 -3408 -2102 -5876 -2538 -922 -3156 -9850 -9236 -938 -7906 -1678 -6402 -8154 -1036 -2950 -5680 -4242 -2088 -5470 -648 -652 -2910 -3642 -4742 -7936 -6576 -4628 -1444 -6844 -9204 -5126 -2376 -1348 -5150 -4794 -3124 -4590 -332 -6134 -4080 -8474 -3498 -2420 -6840 -3618 